lib/systems/default.nix

```diff
@@ -261,23 +261,7 @@ let
       let
         selectEmulator = pkgs:
           let
-            qemu-user = pkgs.qemu.override {
-              smartcardSupport = false;
-              spiceSupport = false;
-              openGLSupport = false;
-              virglSupport = false;
-              vncSupport = false;
-              gtkSupport = false;
-              sdlSupport = false;
-              alsaSupport = false;
-              pulseSupport = false;
-              pipewireSupport = false;
-              jackSupport = false;
-              smbdSupport = false;
-              seccompSupport = false;
-              tpmSupport = false;
-              capstoneSupport = false;
-              enableDocs = false;
+            qemu-user = pkgs.qemu-user.override {
               hostCpuTargets = [ "${final.qemuArch}-linux-user" ];
```

Comment:
Do we want to keep this?  If we just used normal pkgs.qemu-user, we'd just need one Hydra-built binary for emulating any Linux platform.

Reply:
I thought I might get asked extra questions if I removed this. ;) Removed.

(Doing so does increase the necessary download/disk size for only enabling one/a handful of targets. If that's a concern, I could split outputs per architecture for qemu-user (and make the main `out`put a bunch of symlinks).)